src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -162,6 +162,9 @@
   <data name="Remove_Unnecessary_Cast" xml:space="preserve">
     <value>Remove Unnecessary Cast</value>
   </data>
+  <data name="Declare_as_Nullable" xml:space="preserve">
+    <value>Declare as Nullable</value>
```

Comment:
unrelated: I see the casing is quite inconsistent between features. For example "Remove Unnecessary Cast" above and "Cast is redundant" below.